atomsampleviewer_asset_files.cmake

```diff
@@ -46,8 +46,7 @@ set(FILES
     Scripts/ShadowTest.bv.lua
     Scripts/StreamingImageTest.bv.lua
     Scripts/TransparentTest.bv.lua
-    Scripts/_AutomatedPeriodicTestSuite_.bv.lua
-    Scripts/_AutomatedReviewTestSuite_.bv.lua
+	Scripts/_AutomatedReviewWARPTestSuite_.bv.lua
```

Comment:
this seems to have a tab character for indent. not sure if that's an issue

Reply:
ill convert to spaces